packages/server/src/api/controllers/query/validation.ts

```diff
@@ -5,9 +5,9 @@ const OPTIONAL_STRING = Joi.string().optional().allow(null).allow("")
 
 export function queryValidation() {
   return Joi.object({
-    _id: Joi.string(),
-    _rev: Joi.string(),
-    name: Joi.string().required(),
+    _id: OPTIONAL_STRING,
+    _rev: OPTIONAL_STRING,
+    name: OPTIONAL_STRING,
```

Comment:
Is this changing from required to optional on purpose?

Reply:
Good catch - we recently have been unifying the `QueryPreview` and `Query` objects as there is very little difference between the two, but this is a valid difference.

A query preview can be sent without a name, but a query cannot be saved without a name.

I've update the code to reflect this.